_episodes/03-sql-joins-aliases.md

```diff
@@ -8,6 +8,13 @@ objectives:
 - "Employ joins to combine data from two tables."
 - "Apply functions to manipulate individual values."
 - "Employ aliases to assign new names to items in a query."
+keypoints:
+- "Use the `JOIN` command and `ON` or `USING` keywords to combine data from two tables."
```

Comment:
Yeah that's clearer phrasing. But I included both JOIN and ON because using JOIN w/o ON is atypical and not recommended (the cross join behavior does not even occur in all SQLs). Maybe, to make clear they're linked:

- "Use the JOIN command to combine data from two tables---the ON or USING keywords specify which columns link the tables."